systems/plants/setupLCPmex.cpp

```diff
@@ -0,0 +1,275 @@
+#include "mex.h"
+#include <iostream>
+#include "drakeUtil.h"
+#include "RigidBodyManipulator.h"
+#include "math.h"
+#include <sstream>
+
+using namespace Eigen;
+using namespace std;
+
+template <typename DerivedA, typename DerivedB, typename DerivedC, typename DerivedD> 
```

Comment:
I've been moving towards giving the template arguments more distinctive name, e.g. DerivedA --> DerivedQ, DerivedB --> DerivedV, etc.